packages/core/src/test/codewhisperer/commands/basicCommands.test.ts

```diff
@@ -75,35 +74,10 @@ describe('CodeWhisperer-basicCommands', function () {
         sinon.restore()
     })
 
-    it('test get()', async function () {
-        const fakeMemeto = new FakeMemento()
-        await fakeMemeto.update(CodeWhispererConstants.autoTriggerEnabledKey, true)
```

Comment:
These cw get/set functions don't do anything except call globalState. So I removed them and these tests (which aren't testing anything useful).